### DISCUSSION TITLE
提示“需要管理员批准”

### DISCUSSION THREAD

**Discussion by txyyh** [spencerwooo/onedrive-vercel-index#227] [Q&A] [answered]
本人是 A1 用户，本身不是全局管理员

请问我是必须要修改 client_ID 和 **client_secret** 吗

![image](https://user-images.githubusercontent.com/41893946/147853719-1013d4f6-933d-4f34-ab98-b0b03d623b06.png)

**Comment by txyyh** [comment, marked as answer]
[#215 (reply in thread)](https://github.com/spencerwooo/onedrive-vercel-index/discussions/215#discussioncomment-1892784)

**Comment by spencerwooo** [comment]
看样子是这样了，如果你不是管理员的话得自己生成自己的了
